website/docusaurus.config.js

```diff
@@ -6,12 +6,12 @@ const isDeployPreview = process.env.CONTEXT === 'deploy-preview';
 const i18nConfig = {
   defaultLocale: 'en',
   locales: isDeployPreview ? ['en'] : [
-    'ar',    'cs',    'de',    'en',
-    'es-ES', 'fil',   'fr',    'gl',
-    'hi',    'it',    'ja',    'ko',
-    'pl',    'pt-BR', 'ro',    'ru',
-    'sr',    'sr-CS', 'tg',    'vi',
-    'yo',    'zh-CN', 'zh-TW'
+    "ar-SA", "cs-CZ", "de-DE", "es-ES", "fil-PH",
```

Comment:
I might disable some languages, I'll check which does not worth to enable it.

Reply:
I think we should enable:

- es
- ch-tw
- cz
- fr
- de
- it
- po
- pt
- ru
- sr
- vi
- yo

Reply:
Agree with the list.